web/pages/onboarding.tsx

```diff
@@ -49,13 +244,57 @@ export default function Onboarding() {
     }
   }
 
+  let answersState = [];
+  for (let i = 0; i < questionData.length; i++) {
+    if (questionData[i].dropdown) {
+      let ddAns = Array(questionData[i].options.length).fill(false);
+      answersState.push(ddAns);
+    } else {
+      answersState.push(Array(questionData[i].options.length).fill(false));
+    }
+  }
```

Comment:
nit: rewrite using JS array methods for readability :)) Consider doing this for all your for-loop implementations